### ISSUE TITLE
Write tutorials for `align` module

### ISSUE THREAD

**Issue by bdpedigo** [microsoft/graspologic#414]
`graspy.align` is being added by #410. 

That PR will not have a tutorial. 

Need to add a notebook (or multiple notebooks) with simple and clear demonstration of functionality in that module.

**Comment by alyakin314** [comment]
this all true.

(related / cross-listed with #401 .)

**Comment by bdpedigo** [comment]
just a note - i agree is related to #401. But I imagine having one set of basic tutorials for align module, which operate on simulated data (simulated in R^d). 

then i imagine having a more extensive tutorial which describes the alignment in the context of the latent distribution test and why it matters for that (which would be part of what is listed here) 

@alyakin314 agree/disagree?

**Comment by caseypw** [comment]
@bdpedigo I'd be up for writing tutorials for this for Sprint 1 as well. I'm interested in writing these tutorials or tutorials for the clustering algorithms.

**Comment by Jingyan230** [comment]
I'm interested in this issue too. I think maybe writing the tutorials will be a more applicable task for me, since I'm just starting to learn the graph theory. Not sure, need more advice.

**Comment by alyakin314** [comment]
of some relevance: 
https://nbviewer.jupyter.org/url/alyakin314.github.io/assets/seedless_1.ipynb
of less relevance:
https://nbviewer.jupyter.org/url/alyakin314.github.io/assets/seedless_2.ipynb

**Comment by Jingyan230** [comment]
@bdpedigo  I would like to take this one.

**Comment by bdpedigo** [comment]
new PR https://github.com/microsoft/graspologic/pull/419

**Comment by Jingyan230** [comment]
Aims for the tutorial:

1. Sign flips.
- Explain the function of the module.
- Write an example of using the module to move two datasets to the same orthant.

2. Orthogonal Procrustes
- Explain the function.
- Write an example of  using Orthogonal Procrustes for matching two datasets.

3. Seedless Procrustes
- Explain the function.
- Write an example of using Seedless Procrustes for embedding two different graphs.

**Comment by alyakin314** [comment]
@Jingyan230 
align now exists (#419), feel free to hit me up if you have any questions regarding it.

**Comment by Jingyan230** [comment]
Here are some simple examples for tutorial.
Simple rotation examples (Y was built from rotating X initially.)
https://nbviewer.jupyter.org/github/Jingyan230/notebook/blob/my-feature/Aligning_tutorial_sample_F1.ipynb
https://nbviewer.jupyter.org/github/Jingyan230/notebook/blob/my-feature/Aligning_tutorial_sample_F1_90.ipynb
Same shape example (n_X =n_Y =10)
https://nbviewer.jupyter.org/github/Jingyan230/notebook/blob/my-feature/Aligning_tutorial_sample_F2.ipynb
Different shape example (n_X=10, n_Y=20)
https://nbviewer.jupyter.org/github/Jingyan230/notebook/blob/my-feature/Aligning_tutorial_sample_F3.ipynb
@bdpedigo  @alyakin314 Please tell me if you have suggestions on how to improve them.

**Comment by alyakin314** [comment]
@Jingyan230 are you saying that these are just examples you want to use, or these are versions of tutorial notebooks? i.e. should i give feedback only on the models you generate the data from, or everything?

**Comment by alyakin314** [comment]
assuming you were just providing this as examples of the data:
1. sign flips is improper to use in a setting other than literally when dimensions are flipped (i.e. some reflections around coordinate axis), i.e. one shouldn't use them for reflections. it just so happens that 180 degrees rotation overlaps with the reflections around both x-axis and y-axis.
2. what's up with this rotation code? why don't you just multiply by a rotation matrix? that's a one-line thing. 
3. sort of confused by what's happening in the n_1 = 10, n_2 = 20.. what are all these points? why are they not overlapping? any reason you didn't don't like what's in here: https://nbviewer.jupyter.org/url/alyakin314.github.io/assets/seedless_1.ipynb ? this also yields n_2 = 2 * n_1...
4. having the exactly same data in X and Y up to rotation _is_ a setting where alignment will work, but not _isn't the only_ setting where it works, because it aligns distributions. i think it's important to make it clear, so i'd say it'd be nice to have an example where the X and Y are independent samples from the same distribution up to the rotation. e.g.:
```
X = MVN([0.5, 0.5], 0.05 * I)
Y = MVN([0.5, 0.5], 0.05 * I) @ W 
```
but you can probably come up with a smarter example than that.

assuming you are saying that these are tutorials:
5. please use markdown rather than comments for text. (honestly, even if you're just providing it to us as a sketch, that'd be nice becuase it's much easier to read.)
6. the tutorials should be split by module, not by example.
7. explanations are as important as tutorials as examples are.

basically just see other tutorials. e.g. https://graspy.neurodata.io/tutorials/embedding/adjacencyspectralembed

**Comment by alyakin314** [comment]
*by module i meant by class

**Comment by Jingyan230** [comment]
@alyakin314  Thanks for the advice!
